### ISSUE TITLE
Question about Single Model Ablation Study on CoNLL 2014 Test Data Set in the paper

### ISSUE THREAD

**Issue by ghost** [zhawe01/fairseq-gec#16]
I experimented with your program and data. However, I can not get the result like the paper.
What kind of system does Transformer of the item of Model include?

**Comment by zhawe01** [comment]
Can you provide more details about your results?

**Comment by ghost** [comment]
Thank you for your reply . I used your data and programs. And I used train.sh for training . At that time , I referred to the Train without the pre-trained model.  Evaluation used generate.sh. As a result of experiment , F-scorer was 48 . In table5 of your paper, I recognize that it is a transformer and its accuracy is 48 and copying is 54. In this case, I think that the accuracy should be 54. 
Please give me some advice

**Comment by zhawe01** [comment]
Did you use the spell corrected test.src.

**Comment by ghost** [comment]
> Thank you for your reply . I used your data and programs. And I used train.sh for training . At that time , I referred to the Train without the pre-trained model. Evaluation used generate.sh. As a result of experiment , F-scorer was 48 . In table5 of your paper, I recognize that it is a transformer and its accuracy is 48 and copying is 54. In this case, I think that the accuracy should be 54.
> Please give me some advice

Hello, may I ask have you fix that problem? I meet the slimiar problem like yours. I used train.sh for training without pre-train model and bash generate.sh, as a result, F0.5 only achieve 38, I wonder what's test.m2 file you used?

**Comment by ghost** [comment]
> Did you use the spell corrected test.src.

Thank you for your reply, but i still can't figure out what's spell corrected test.src.  After training for 15 epoches, I download test.m2 file and bash g.sh directly without changing any params, but my result (F0.5 = 38) can't achieve that show on the paper, can you give me some advice ?
